parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -103,9 +103,8 @@ downloadCrictl() {
 installCrictl() {
     CPU_ARCH=$(getCPUArch)  #amd64 or arm64
     currentVersion=$(crictl --version 2>/dev/null | sed 's/crictl version //g')
-    local CRICTL_VERSION=${KUBERNETES_VERSION%.*}.0
-    if [[ ${currentVersion} =~ ${CRICTL_VERSION} ]]; then
-        echo "version ${currentVersion} of crictl already installed. skipping installCrictl of target version ${CRICTL_VERSION}"
+    if [[ "${currentVersion}" == "" ]]; then
```

Comment:
This reads like we're not installing if currentVersion is empty. Which doesn't seem like the goal. Do we want != ?

Reply:
oops. flipped